element_calcium_imaging/field_processing.py

```diff
@@ -293,18 +293,21 @@ def make(self, key):
             ops_path = find_full_path(processed_root_data_dir, extra_params["ops_path"])
 
             # calculate aspect ratio and diameter
-            um_height, um_width = (scan.ScanInfo.Field & key).fetch1(
-                "um_height", "um_width"
-            )
-            aspect = um_height / um_width
+            px_height, px_width, um_height, um_width = (
+                scan.ScanInfo.Field & key
+            ).fetch1("px_height", "px_width", "um_height", "um_width")
+            aspect = (um_height / px_height) / (um_width / px_width)
```

Comment:
Could you explain the logic here? Why do we need this change to calculate the aspect ratio?